### PR TITLE
Add interactive (Cookie notice)

### PR DESCRIPTION
**Pull request by ryyppy**
How to test (in Chrome) after Accept:
- Open Devtools >> Application
- Select Cookies tab in sidebar
- Delete the value `gdpr_accept`
- Refresh page (now you should be able to see the message again)

### CODE REVIEW

**Comment by ryyppy** [comment]
@okonet @cezarneaga The technical part is done now, you can take this branch for styling etc if you want

File: common/cookie.re [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+  let read = (content, name) => {
+    /* Note: This is a hacky approach of doing imperative programming.
+       I was literally translating a very imperative example from JS
+       to Reason. There are ways to rewrite it in a functional way which
+       may be much cleaner */
+
+    open Js.String;
+    let nameEQ = name ++ "=";
+    let ca = split(";", content);
```
Comment (cezarneaga) [inline common/cookie.re]:
```suggestion
    let ca = split("; ", content);
```

Comment (ryyppy) [reply common/cookie.re]: Not sure why this is important? we trim all tokens in line 55 anyways?

Comment (cezarneaga) [reply common/cookie.re]: sorry. did not see that. cleaner too.  :)

**Pull request merged**